opsdroid/events.py

```diff
@@ -413,6 +415,44 @@ def __init__(self, name=None, params=None, *args, **kwargs):  # noqa: D107
         self.room_params = params or {}
 
 
+class Video(File):
+    """Event class specifically for video files."""
+
+    async def get_bin(self):
+        """ Return the binary representation of video """
+
+        """Get a bitarray of the video bytes.
+        This method enable video bytes to be converted to hex/bin.
+        Doc: https://github.com/scott-griffiths/bitstring/blob/master/doc/bitarray.rst
+        """
+        fbytes = await self.get_file_bytes()
+        my_bit_array = BitArray(fbytes)
+
+        # Return that bitarray
+        return my_bit_array.bin
+
+    async def get_properties(self):
+        """Get the video properties like codec, resolution.
+        Returns Video properties saved in a Dictionary"""
+
+        # get bytes of file
+        fbytes = await self.get_file_bytes()
+
+        # create a temp_vid file to store the bytes
+        temp_vid = tempfile.NamedTemporaryFile(prefix="opsdroid_vid_")
```

Comment:
Does this temp file get removed after we have done what we needed with it?

Reply:
> Hello,
> 
> I've added a few suggestions and questions on the code. I need to think about what would be best on the test side. If getting ffmpeg installed or maybe use a mock there thinking
> 
> Also, since ffmpeg is a requirement to make this work, can you update the documentation to mention this? We should add it there so users know that this is expected otherwise people will only know they need ffmpeg after the first failure.

Yeah sure i will make changes to the documentation! can you help me where should i do that?

And for ubuntu you can add the 3 lines (mentioned above) to install ffmpeg in tt, the main problem is for windows since its hard to install ffmpeg using cmd line there.

Reply:
it does after returning the value, but i have added a line to manually deleting that temp file after returning the value in next commit.